### PR TITLE
feat(test): Enzyme refactor part 3

### PR DESCRIPTION
**Pull request by Fewwy**
Refactoring the last enzyme tests

Current issues:

### CODE REVIEW

File: config/rtlwrapper.js [hunk 1]
```diff
@@ -13,3 +14,13 @@ export const mountWithRouterAndProvider = (Component, store) => {
         </Provider>
     );
 };
+
+export const mountWithRouterAndProviderAndIntl = (Component, store) => {
+    return render(
+        <IntlProvider>
+            <Provider store={store}>
+                <BrowserRouter>{Component}</BrowserRouter>
```
[the remark this answers is not in the document]
Comment (Fewwy) [reply config/rtlwrapper.js:22]: I don't think so.
I tried to use it and it was throwing errors.

File: src/PresentationalComponents/PatchSetWrapper/PatchSetWrapper.test.js [hunk 2]
```diff
@@ -34,52 +32,31 @@ const testProps = {
...
         };
-        const wrapper = mountWithIntl(<Provider store={store}><PatchSetWrapper {...testHiddenState} /></Provider>);
-        expect(wrapper.find(UnassignSystemsModal).exists()).toBeFalsy();
-    });
-    it('should props propagate to child components', () => {
-        const wrapper = mountWithIntl(<Provider store={store}><PatchSetWrapper {...testProps} /></Provider>);
-        const unassignSystemsModalProps = wrapper.find(UnassignSystemsModal).props();
-        const patchSetWizardProps = wrapper.find(PatchSetWizard).props();
-
-        expect(unassignSystemsModalProps).toEqual(
```
[the remark this answers is not in the document]
Comment (Fewwy) [reply src/PresentationalComponents/PatchSetWrapper/PatchSetWrapper.test.js:65]: As far as I understand testing props is against RTL "vision", we should test it from the point of view of the user.
The user doesn't know about props or shouldn't care about them.
We should create a separate test to test the hook or use Cypress later to test a fully rendered UI.

File: config/rtlwrapper.js [hunk 3]
```diff
@@ -13,3 +14,13 @@ export const mountWithRouterAndProvider = (Component, store) => {
         </Provider>
     );
 };
+
+export const mountWithRouterAndProviderAndIntl = (Component, store) => {
```
[the remark this answers is not in the document]
Comment (Fewwy) [reply config/rtlwrapper.js:18]: But it's not a React Context so I don't this name is appropriate either

[the remark this answers is not in the document]
Comment (mkholjuraev) [reply config/rtlwrapper.js:18]: Yeah, I understand your point. My argument is that context does not only mean React context. Rather it means all the things that component is made aware of.

**Review by mkholjuraev — APPROVED** [review]
I went through again, and I feel getting this PR merged not only unblocks the Patch app migrations, but also the tests are improved in some sense. @Fewwy Let's get this merged.

**Pull request merged**